PC/launcher.c

```diff
@@ -1412,22 +1412,31 @@ show_help_text(wchar_t ** argv)
 Python Launcher for Windows Version %ls\n\n", version_text);
     fwprintf(stdout, L"\
 usage:\n\
-%ls [launcher-args] [python-args] script [script-args]\n\n", argv[0]);
+%ls [launcher-args] [python-args] [script [script-args]]\n\n", argv[0]);
     fputws(L"\
 Launcher arguments:\n\n\
 -2     : Launch the latest Python 2.x version\n\
 -3     : Launch the latest Python 3.x version\n\
 -X.Y   : Launch the specified Python version\n", stdout);
     if (canDo64bit) {
         fputws(L"\
-     The above all default to 64 bit if a matching 64 bit python is present.\n\
+     The above all default to 64 bit if a matching 64 bit python is present*.\n\
 -X.Y-32: Launch the specified 32bit Python version\n\
 -X-32  : Launch the latest 32bit Python X version\n\
 -X.Y-64: Launch the specified 64bit Python version\n\
 -X-64  : Launch the latest 64bit Python X version", stdout);
     }
     fputws(L"\n-0  --list       : List the available pythons", stdout);
     fputws(L"\n-0p --list-paths : List with paths", stdout);
+    fputws(L"\n\n If no script is specified the specified interpreter is opened.", stdout);
+    fputws(L"\nIf an exact version is not given, using the latest version can be overridden by", stdout);
+    fputws(L"\nany of the following, (in priority order):", stdout);
+    fputws(L"\n An active virtual environment", stdout);
+    fputws(L"\n A shebang line in the script (if present)", stdout);
+    fputws(L"\n Matching PY_PYTHON2 or 3 Enviroment variable", stdout);
```

Comment:
Without reading the docs, I'd be inclined to think that `PY_PYTHON3` should override the latest Python 3 selection, but it only applies with the `-3` command-line option. The same for `PY_PYTHON2`. So maybe change this to 

> "A PY_PYTHON2 or PY_PYTHON3 environment variable in combination with the -2 or -3 argument, respectively."

Reply:
Modified to:
 With -2 or -3 flag a matching PY_PYTHON2 or PY_PYTHON3 Enviroment variable

_(Shorter than suggested but clearer than before)_.